### PR TITLE
MGDAPI-5245 Test case updates for 1.32.0

### PR DESCRIPTION
**Pull request by trepel**
# Issue link

https://issues.redhat.com/browse/MGDAPI-5245

# What

Standard updates of test cases based on 1.32.0 testing

# Verification steps

Eye review only.

### CODE REVIEW

File: test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ cd test/scripts/backup-restore
 NS_PREFIX=redhat-rhoam ./j07-verify-rhsso-backup-and-restore.sh | tee test-output.txt
 ```
 
-4. **STS** - Reach out to SRE/QE for the `ManagedOpenShift-Support-Role` credentials in order to run the backup and restore script
+4. **STS** - Reach out to QE for the `osdCcsAdmin` credentials in order to run th
```
Comment (martinhesko) [inline test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md:86]: Probably accidentally cut off the rest of the sentence here.

**Comment by martinhesko** [comment]
Verified that all of the test cases updated were performed, all the changes look fine, only the 1 commented section needs changing.

File: test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md (hunk 1 above)
Comment (trepel) [reply test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md:86]: Good catch! Will force fix

Comment (trepel) [reply test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md]: Done

**Pull request merged**